tests/test_business.py

```diff
@@ -170,3 +225,154 @@ def test_equality(self):
 
         assert bmd1 != bmd3
         assert hash(bmd1) != hash(bmd3)
+
+
+class TestBusinessIntro(TestBusinessBase):
+    def test_slots(self, business_intro):
```

Comment:
```suggestion
    def test_slot_behaviour(self, business_intro):
```
we have consistent naming so

Reply:
Done.